src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -2239,7 +2240,7 @@ export default class AircraftModel {
                 offsetGameTime
             ]);
             // TODO: this can be abstracted
-        } else if (abs(offsetGameTime - this.relativePositionHistory[this.relativePositionHistory.length - 1][2]) > 4 / GameController.game_speedup()) {
+        } else if (abs(offsetGameTime - this.relativePositionHistory[this.relativePositionHistory.length - 1][2]) > 4000 / GameController.game_speedup()) {
```

Comment:
Do we know what `4` means in this context? I can't really understand the original code lol.

Reply:
Yea, I thought the same, but I investigated and it was time in seconds, and is now time in milliseconds:)